lib/utils/show_settings_dialog.dart

```diff
@@ -491,9 +491,15 @@ class SettingsDialogState extends ConsumerState<SettingsDialog> {
                               (value) {
                                 ref.read(cleanseProvider.notifier).state =
                                     value;
-
                                 _saveToggleStates();
                               },
+                              '''
```

Comment:
Have a look at how the tooltip strings are constructed in other places and please conform to that standard. Thanks.